.github/workflows/checks.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
     # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
     strategy:
       matrix:
-        msrv: [1.63.0]
+        msrv: [1.70.0]
```

Comment:
This bumps minimal required version to `newest - 1`. I'd keep it as it is until we introduce code that is not compatible with `1.63`.

Reply:
errata: this comment was intended for `rust-version` change in Cargo.toml

Reply:
Above comment applies to `Cargo.toml` primarily. This one should probably test major revisions from 1.63 to `newest -1`